lib/webmock/request_body_diff.rb

```diff
@@ -15,9 +15,10 @@ def body_diff
       HashDiff.diff(request_signature_body_hash, request_stub_body_hash)
     end
 
-    private
-
     attr_reader :request_signature, :request_stub
+    private :request_signature, :request_stub
+
+    private
```

Comment:
Declaring `attr_reader` after `private` is [no longer a warning](https://bugs.ruby-lang.org/issues/10967) as of Ruby 2.3, so this is a legacy workaround. Retaining it may or may not be worthwhile to prevent warnings in old versions of ruby.